<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateDeckDialog</class>
 <widget class="QDialog" name="CreateDeckDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:url(:/img/img/menuBg.png)</string>
  </property>
  <widget class="QPushButton" name="validateButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190, 190, 190)</string>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QLabel" name="weightingLabel">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>480</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background:Transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;/20&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190, 190, 190)</string>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>480</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(190, 190, 190)</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLabel" name="nbrOfCardsLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>480</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:Transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; color:#ffffff;&quot;&gt;0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDlabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; text-decoration: underline; color:#ffffff;&quot;&gt;ID de la carte :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="cardID">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>480</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>884</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>6</number>
   </property>
   <widget class="QWidget" name="first">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="upF"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downF"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="second">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="upS"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downS"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="third">
    <attribute name="title">
     <string>Tab 3</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="upT"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downT"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="fourth">
    <attribute name="title">
     <string>Tab 4</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <layout class="QHBoxLayout" name="upFo"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downFo"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Tab 5</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <layout class="QHBoxLayout" name="upFi"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downFi"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Tab 6</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="upSix"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downSix"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Tab 7</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_8">
     <item>
      <layout class="QHBoxLayout" name="downSeven"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="upSeven"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Tab 8</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_9">
     <item>
      <layout class="QHBoxLayout" name="upE"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downE"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Tab 9</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="upN"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downN"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Tab 10</string>
    </attribute>
    <layout class="QVBoxLayout" name="verticalLayout_11">
     <item>
      <layout class="QHBoxLayout" name="upTen"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="downTen"/>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
